WordPress/src/main/java/org/wordpress/android/ui/main/MeViewModel.kt

```diff
@@ -32,14 +33,18 @@ class MeViewModel
     @Named(BG_THREAD) val bgDispatcher: CoroutineDispatcher,
     private val selectedSiteRepository: SelectedSiteRepository,
     private val recommendApiCallsProvider: RecommendApiCallsProvider,
-    private val analyticsUtilsWrapper: AnalyticsUtilsWrapper
+    private val analyticsUtilsWrapper: AnalyticsUtilsWrapper,
+    val unifiedAboutFeatureConfig: UnifiedAboutFeatureConfig
```

Comment:
If we're only using `unifiedAboutFeatureConfig` in the fragment, maybe we should inject it there directly, and not here. (Ideally, it should be here along with the logic on what to show/hide, but for now, I think it's less confusing to just leave it on the fragment. Let me know if you disagree.)

Reply:
I had it in the fragment to begin with, then moved to viewmodel.  Looks like injecting in fragment seems to be an older practice and everything is done in viewmodel now.

Reply:
>Looks like injecting in fragment seems to be an older practice and everything is done in viewmodel now.

That's true, but in this case, we're moving something to the view model while only using it in the fragment. Don't you agree that can be confusing? The way I see it, if we're moving that field into the view model, then we should also move the logic that uses it. Otherwise, I don't see any reason to do so, and it should probably stay in the fragment where it's being used (which is the approach I'd go for here). 

It's a matter of code readability, so it might be subjective, but I believe it's good to [avoid indirections](https://matthewrocklin.com/blog/work/2019/06/23/avoid-indirection) like these.